book/forms.rst

```diff
@@ -1122,6 +1122,24 @@ the choice is ultimately up to you.
             ));
         }
 
+.. caution::
+
+    When the name of your form class matches any registered field type that has
+    its own theme fragment, your form won't be rendered correctly. A form
+    ``AppBundle\\Form\\Type\\PasswordType`` to change a user password for
+    instance will be mistaken for the built-in ``PasswordType`` field type and
+    will be rendered incorrectly using the ``password_widget`` theme fragment.
+    You can still use the class name of your choice by overriding the
+    ``getBlockPrefix`` method of your form class::
+
+        public function getBlockPrefix()
+        {
+            return 'app_bundle_password';
+        }
+
+    Read the chapter ":doc:`/cookbook/form/create_custom_field_type.html`" for
```

Comment:
You need to omit the actual extension (`.html`) here. Sphinx will take care of it when generating the documentation.